src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -186,6 +188,10 @@ protected function refreshUser(TokenInterface $token)
                         $this->logger->debug('Cannot refresh token because user has changed.', ['username' => $refreshedUser->getUsername(), 'provider' => \get_class($provider)]);
                     }
 
+                    if (null !== $this->dispatcher) {
+                        $this->dispatcher->dispatch(new LogoutOnChangeEvent($token, $refreshedUser), SecurityEvents::LOGOUT_ON_CHANGE);
```

Comment:
is the name correctly describing the dispatch point? can we find a more descriptive name?

Reply:
According to https://github.com/symfony/symfony/pull/30955#issuecomment-480780775, we should not use a name but simply the event class name.

Reply:
maybe UserHasChangedEvent ?